src/components/Datagrid/customCells/cells.ts

```diff
@@ -40,6 +40,24 @@ export function readonlyTextCell(
   };
 }
 
+export function tagsCell(
+  tags: Array<{ tag: string; color: string }>,
+  selectedTags: string[],
+  opts?: Partial<GridCell>,
```

Comment:
**suggestion:** One object instead of 3 distinct parameters would make this function more readable